src/components/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import MuiThemeProvider from '@material-ui/core/styles/MuiThemeProvider';
+import { 
+	BrowserRouter as Router,
+	Route,
+} from 'react-router-dom';
+import useStoreon from 'storeon/react'
+import { routes } from "../routes/routes";
+import Loader from './common/Loader';
+import SecureRoute from './common/SecureRoute';
+
+const App = () => { 
+	const { loading } = useStoreon('loading');
+	const { user } = useStoreon('user');
+	return (
+	<React.Fragment>
```

Comment:
Since we don't have multiple child elements ,we can remove the Fragment from here.

Reply:
done